netdisco/ssdp.py

```diff
@@ -10,7 +10,7 @@
 
 from netdisco.util import etree_to_dict, interface_addresses
 
-DISCOVER_TIMEOUT = 5
+DISCOVER_TIMEOUT = 2
 SSDP_MX = 3
```

Comment:
Please decrease this value. As per [the spec](http://www.upnp.org/specs/arch/UPnP-arch-DeviceArchitecture-v1.0-20080424.pdf):

![image](https://user-images.githubusercontent.com/1444314/30722276-3bbfed5e-9ee5-11e7-9040-9f8857c2e2f8.png)

Reply:
Thanks for the hint. I made it be assigned from DISCOVER_TIMEOUT to avoid issues with future tweaking, and added a comment.